docs/scripts/script.min.js

```diff
@@ -53,6 +53,15 @@ class Evolution {
         return shiny;
     }
 }
+function ToggleVisibility(id) {
+    //alert("Toggling Visibility")
+    var x = document.getElementById(id);
+    if (x.style.display === "none") {
+        x.style.display = "grid";
+    } else {
+        x.style.display = "none";
+    }
+} 
```

Comment:
Can you please revert all the changes made to the `docs/` directory, this is only updated when we push to the live website.

Reply:
Gotcha. Leave it with me.